packages/smooth_app/lib/main.dart

```diff
@@ -18,26 +19,23 @@ import 'package:smooth_app/themes/theme_provider.dart';
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  await Sentry.init(
-    (SentryOptions options) {
-      options.dsn =
-          'https://22ec5d0489534b91ba455462d3736680@o241488.ingest.sentry.io/5376745';
-    },
-  );
+  final PackageInfo packageInfo = await PackageInfo.fromPlatform();
+
   /* TODO: put back when we have clearer ideas about analytics
   await MatomoTracker().initialize(
     siteId: 2,
     url: 'https://analytics.openfoodfacts.org/',
   );
    */
-  try {
-    runApp(const SmoothApp());
-  } catch (exception, stackTrace) {
-    await Sentry.captureException(
-      exception,
-      stackTrace: stackTrace,
-    );
-  }
+
+  await SentryFlutter.init(
+    (SentryOptions options) {
+      options.dsn =
+          'https://22ec5d0489534b91ba455462d3736680@o241488.ingest.sentry.io/5376745';
+      options.sentryClientName = 'sentry.dart.smoothie/${packageInfo.version}';
```

Comment:
What does this do?

Reply:
> /// Sentry client name used for the HTTP authHeader and userAgent eg
> /// sentry.{language}.{platform}/{version} eg sentry.java.android/2.0.0 would be a valid case

Since I was unsure if it only affects the sentry http client or is also used to filter bugs by app version or something I took a look at the implementation and it seems like this value is not used at all. Needs further investigation

Reply:
https://github.com/getsentry/sentry-dart/issues/584#issuecomment-917378211

Reply:
Thanks! I personally lean towards omitting code that makes no difference, but feel free to use your judgement here.